static/src/javascripts/projects/common/modules/atoms/quiz.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'bean',
+    'common/utils/$',
+    'common/utils/fastdom-promise'
+], function (
+    bean,
+    $,
+    fastdom
+) {
+    return {
+        handleCompletion: function () {
+            if (HTMLFormElement.prototype.checkValidity) {
+                var $quiz = $('.js-atom-quiz--instant-reveal');
+                if ($quiz.length > 0) {
+                    bean.on(document, 'click', Array.prototype.slice.call($quiz), function (e) {
+                        var quiz = e.currentTarget;
+                        if (quiz.checkValidity()) {
+                            var $bucket__message = null,
+                                bucket = $(':checked + .atom-quiz__answer__item--is-correct', quiz).length;
+                            do {
+                                $bucket__message = $('.js-atom-quiz__bucket-message--' + bucket, quiz);
```

Comment:
here it's actually iterating through a list of potential elements based on your score:

```
<div class="js-atom-quiz__bucket-message--5">nice</div>
<div class="js-atom-quiz__bucket-message--4">ok</div>
```
etc.

we can't know what the buckets and total etc will be